build.assets/Makefile

```diff
@@ -446,18 +446,26 @@ define RELEASE_RECIPE
 			-v "/tmp:/tmp" \
 			-v "$(WORKSPACE):/go/teleport" \
 			-w /go/teleport \
-			-u 1000:1000 \
```

Comment:
Why was this removed?

Reply:
I had the problems with local fs permissions. 1000:1000 did not match the local user/group. You know, let me discard all changes to Makefile here. I guess, those docker flags should be on the GHA side anyway.